### DISCUSSION TITLE
Feature request: web UI that can join the session so participants don't have to install software

### DISCUSSION THREAD

**Discussion by thgh** [mistweaverco/bananas#63] [Ideas]
The host would have to fill in the URL of the webserver which is then reflected in the session URL. Maybe even allow pretty urls per host.

**Comment by gorillamoe** [comment]
Can be done with some limitations. If you join via web, you can't share your screen and have cursors drawn on your screen. Also no way of being able to give control to your mouse or keyboard (planed for v2).

**Comment by DegrangeM** [comment]
There is the screencapture api https://developer.mozilla.org/en-US/docs/Web/API/Screen_Capture_API/Using_Screen_Capture

**Comment by gorillamoe** [comment]
I know, I did not mean can't stream at all, just that the catch is that the host on the web can't have remote Cursors drawn on the screen.

**Comment by gorillamoe** [comment]
Moved this to discussions, because it's not yet on the roadmap, but if it get's enough up votes, it'll get its place probably in v2
